src/Documentation/doc/advanced/corporation/faq.md

```diff
@@ -0,0 +1,211 @@
+
+> How much boost materials that I should buy?
+
+Check the optimizer in this [section](./boost-material.md).
+
+> Why does setAutoJobAssignment not take effect immediately?
+
+It only takes effect in next cycle's START state.
+
+> Why does energy and morale matter?
+
+They are used for calculating employeeProductionByJob, then that property is used for calculating other things: RP, material's quality, product's stats, division raw production, material/product's MaxSalesVolume.
+
+> What does Intern employee do?
+
+Don't bother with that job. Its only purpose is to maintain energy and morale. Tea/party script can do it for you, and it is very simple to implement.
+
+> Everybody tells me to use 1/9 as Intern ratio, but when I use it, energy and morale still drop.
+
+You can only use that ratio when your corporation works fine (funds > 0 or profit > 0). If it does not, use 1/6.
+
+> Are there any other ways to maintain these 2 stats?
+
+There are researches for that. However, you should never buy them, it's always better spending your RP elsewhere or just stock up on RP.
+
+> Buying tea and throwing party cost me too much money. Why are they so expensive?
+
+Tea and party are cheap. If your budget is so low that they cost you too much money, it means you wasted too much funds.
+
+> How much money should I spend to throw party? How often should I buy tea / throw party?
+
+Check this [section](./office.md).
+
+> How do I know if the qualities of my input materials are too low and need to be improved?
+
+Check this [section](./quality.md).
+
+> What are Awareness and Popularity?
+
+Check this [section](./wilson-analytics-advert.md).
+
+> Should I buy Dream Sense?
+
+No. Check this [section](./wilson-analytics-advert.md) for the reason.
+
+> Is Wilson retroactive?
+
+No.
+
+> Does that mean I should buy Wilson as soon as possible? If yes, then why don't you buy any Wilson in round 1 and 2?
+
+You should buy Wilson as soon as possible, but not too soon. Round 1 and 2 are those cases. Check this [section](./wilson-analytics-advert.md) for details.
+
+> What are Demand and Competition?
+
+Check this [section](./demand-competition.md).
+
+> How much should I spend for "Design investment" and "Marketing investment" when I create new product? How do they affect the product?
+
+They are not too important. It's fine to spend 1% of your current funds for them. Check this [section](./product.md) for details.
+
+> Should I buy Market-TA1?
+
+No, wait for Market-TA2. Market-TA1 is useless.
+
+> When should I buy Market-TA2?
+
+As soon as possible, it greatly increases your profit because it can find the optimal price. However, that research is expensive, it costs totally 75000RP (Hi-Tech R&D Laboratory + Market-TA1 + Market-TA2). Depleting entire RP pool extremely degrades your product rating, so I recommend saving up 150000RP before buying it. Depleting half of the RP pool is acceptable, considering the positive effect of Market-TA2.
```

Comment:
```suggestion
As soon as possible, it greatly increases your profit because it can find the optimal price. However, that research is expensive, it costs a total of 75000RP (Hi-Tech R&D Laboratory + Market-TA1 + Market-TA2). Depleting the entire RP pool extremely degrades your product rating, so I recommend saving up 150000RP before buying it. Depleting half of the RP pool is acceptable, considering the positive effect of Market-TA2.
```